packages/ngspice/build.sh

```diff
@@ -3,8 +3,43 @@ TERMUX_PKG_DESCRIPTION="A mixed-level/mixed-signal circuit simulator"
 TERMUX_PKG_LICENSE="BSD 3-Clause, LGPL-2.1"
 TERMUX_PKG_LICENSE_FILE="COPYING"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=35
+TERMUX_PKG_VERSION=37
 TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/ngspice/ngspice-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=c1b7f5c276db579acb3f0a7afb64afdeb4362289a6cab502d4ca302d6e5279ec
-TERMUX_PKG_DEPENDS="libc++"
+TERMUX_PKG_SHA256=9beea6741a36a36a70f3152a36c82b728ee124c59a495312796376b30c8becbe
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+--enable-xspice
+--enable-cider
+--with-readline=yes
+--enable-openmp
+"
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_EXTRA_HOSTBUILD_CONFIGURE_ARGS="
+--with-x=no
+--enable-cider
+--enable-xspice
+"
+TERMUX_PKG_DEPENDS="libc++, readline, fftw"
 TERMUX_PKG_GROUPS="science"
+
+termux_step_host_build(){
+	cp -r $TERMUX_PKG_SRCDIR/* .
+
```

Comment:
Please use tabs instead of spaces for formatting. See https://github.com/termux/termux-packages/wiki/Coding-guideline#script-formatting

Reply:
I believe it has already tabs:

```
000002e0  5f 68 6f 73 74 5f 62 75  69 6c 64 28 29 7b 0a 09  |_host_build(){..|
000002f0  63 70 20 2d 72 20 24 54  45 52 4d 55 58 5f 50 4b  |cp -r $TERMUX_PK|
00000300  47 5f 53 52 43 44 49 52  2f 2a 20 2e 0a 0a 09 23  |G_SRCDIR/* ....#|
```

Reply:
My bad, it seems like me recently messing around with GitHub UI's editor settings caused this visual misinterpretation that it was a tab. Appologies for that